src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/src/StorageSubSystemTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Add a test for invalid storage subsystems.

Reply:
Added.